TwitchClient/GamesListViewController.swift

```diff
@@ -1,108 +0,0 @@
-//
-//  GamesListViewController.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-import SVProgressHUD
-
-class GamesListViewController: UIViewController, WebserviceManagerDelegate, UITableViewDelegate, UITableViewDataSource {
-
-    
-    @IBOutlet weak var gamesTableView: UITableView!
-    
-    
-    var games = [Game]()
-   
-    var selectedGame = Game()
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
-        
-        self.gamesTableView.isHidden = true
-        SVProgressHUD.show()
-        
-        self.getTopGames()
-        
-    }
-   
-    func getTopGames()
-    {
-        // initialize webservice manager and call the web service request
-        let manager = WebServiceManager()
-        manager.delegate = self
-        manager.getTopGames()
-    }
-    //MARK:- Webservice Manager Delegate
-    func didFinishRetrievingData(sender: WebServiceManager, GamesList: [Game]?, StreamList: [Stream]?, error: Bool) {
-        
-        if error == false{
-            self.gamesTableView.isHidden = false
-            SVProgressHUD.dismiss()
-            
-            self.games = GamesList!.sorted{$0.popularity! > $1.popularity!}
-            self.gamesTableView.reloadData()
-        }
-        else
-        {
-            self.showAlert()
-        }
-       
-    }
-    
-    func showAlert()
-    {
-        let alert = UIAlertController.init(title: "Error", message: "Couldn't retrieve the Games Data at the moment. please try again later", preferredStyle: .alert)
```

Comment:
Even when some of this code goes to the VM, the strings should be in localizable string files, they should never be in the code.

Topics: clean code, localization in Swift.